core/ipns.go

```diff
@@ -134,3 +134,10 @@ func ipnsEntryDataForSig(e *ipnspb.IpnsEntry) []byte {
 	},
 		[]byte{})
 }
+
+func ipnsAPIPathTransform(url, peerID string) string {
+	if !strings.HasSuffix(url, "/") {
+		url = url + "/"
+	}
+	return url + "ob/ipns/" + peerID
+}
```

Comment:
Great to see helpers like this. Let's unit test this low-hanging fruit, too.

Reply:
Done